math/csint.f

```diff
@@ -254,7 +254,7 @@ Subroutine C9INI(ndata,xdata,fdata,dfdata,break,c)
       Do j = 1,ndata
             idx(j) = j
       Enddo
-      Call SVRGP(ndata,xdata,break,1,idx)
```

Comment:
I liked this code better before

Reply:
then should I change the prototype that doesn't match?